component/core/src/main/java/org/exoplatform/social/core/space/impl/SpaceServiceImpl.java

```diff
@@ -408,10 +408,12 @@ public Space createSpace(Space space, String creator, List<Identity> identitiesT
     // Add creator as a manager and a member to this space
     String[] managers = space.getManagers();
     String[] members = space.getMembers();
+    String[] redactors = space.getRedactors();
     managers = (String[]) ArrayUtils.add(managers,creator);
     members = (String[]) ArrayUtils.add(members,creator);
     space.setManagers(managers);
     space.setMembers(members);
+    space.setRedactors(redactors);
```

Comment:
why do you need this modification ?

Reply:
yes, to use it in filter

Reply:
Right now we will not need to add managers on the redactors filters